erpnext/subcontracting/doctype/subcontracting_inward_order/test_subcontracting_inward_order.py

```
@@ -0,0 +1,556 @@
+		rm_in.items[1].qty = 5
+		rm_in.items[2].qty = 12
+		rm_in.submit()
+
+		scio.reload()
+		wo = frappe.get_doc("Work Order", scio.make_work_order()[0])
+		self.assertEqual(wo.qty, 2)
+
+	def test_rm_return(self):
+		from erpnext.stock.serial_batch_bundle import get_batch_nos, get_serial_nos
+
+		so, scio = create_so_scio()
```

Ruff (0.13.3)

Unpacked variable `so` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)